src/containers/SignUpPage.js

```diff
@@ -8,6 +8,8 @@ import { signUp } from 'actions/userActions';
 import SignUpForm from 'components/user/SignUpForm';
 import routes from 'constants/routesPaths';
 
+import demo from '../resources/images/demo3x.png';
```

Comment:
Maybe you can add the resources folder to the resolved paths (you can find this file on `webpack/shared/resolve.js`) so you'll able to use absolute paths instead